README.md

```diff
@@ -58,7 +58,7 @@ How to use this application:
 
 -   Setup website Progressive Web Application:
     -   Go to "_Website Settings_" and add into "_HTML HEADER & ROBOTS_" the web app manifest: `<link rel="manifest" href="/manifest.json">`
-    -   Go to "_Website Settings_" and add into "_BANNER_" the link to trigger PWA install (feel free to customize design): `<a id="pwa-install-link" href="#">PWA</a>`
+    -   Go to `/install` url and press on `install` inside pop-up message
```

Comment:
@AminovE99 This is related **not** related to setup of PWA but actual usage by end users.

```suggestion


After setup has been done, just go to `/install` page and press on `install` inside pop-up message.
```